src/main/finstore/charts/consul/.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,61 @@
+---
```

Comment:
`charts/` dir should not be submitted as declared dependencies will automatically be downloaded and packaged.